pkg/crds/yaml/generated/management.cattle.io_roletemplates.yaml

```diff
@@ -18,7 +18,7 @@ spec:
     schema:
       openAPIV3Schema:
         description: RoleTemplate holds configuration for a template that is used
-          to create roles and clusterRoles for a cluster or project.
+          to create kubernetes native roles and clusterRoles for a cluster or project.
```

Comment:
```suggestion
// RoleTemplate holds configuration for a template that is used to create native Kubernetes roles and cluster roles for a cluster or project.
```

Reply:
I'm happy to change other fields, but not on generated files like this. Please take a look at the source file, and you can leave a comment there.

Reply:
My bad, just noticed that it's generated, resolving.